src/options.rs

```diff
@@ -88,6 +88,20 @@ pub struct AstarteOptions {
     pub(crate) keepalive: std::time::Duration,
 }
 
+impl Debug for AstarteOptions {
+    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
+        f.debug_struct("AstarteOptions")
+            .field("realm", &self.realm)
+            .field("device_id", &self.device_id)
+            .field("credentials_secret", &self.credentials_secret)
```

Comment:
Remove the `credentials_secret` field from the output or obfuscate it, we should avoid logging sensitive data.

Reply:
Done, thanks